src/app/components/filebrowseruss/filebrowseruss.component.ts

```diff
@@ -198,6 +199,8 @@ export class FileBrowserUSSComponent implements OnInit, OnDestroy {//IFileBrowse
         this.showRenameField(this.rightClickedFile) }},
       { text: "Change Owners", action:() => { 
         this.showOwnerDialog(this.rightClickedFile) }},
+      { text: "Tag", action:() => { 
```

Comment:
Personal preference: "Tag" should be "Tag..." to show that it will spawn a modal/additional options